zorka-core/src/main/java/com/jitlogic/zorka/core/integ/netuitive/JvmSystemStatsReport.java

```diff
@@ -12,9 +12,11 @@
 
     /* Logger */
     private final ZorkaLog log = ZorkaLogger.getLog(JvmSystemStatsReport.class);
+    private Java6CpuUsage java6CpuUsage;
```

Comment:
that log might have been static which is not part of this fix. for private helper members, I'd like to keep it in constructor so that if it needs any variable passed from its parent, I can pass it in there.